libphal/phal_exception.H

```diff
@@ -8,19 +8,20 @@
 
 namespace openpower::phal
 {
-//taken from error_info_defs.H, including the header file breaks logging and
-//debug collector as they need to include ekb in include path
-enum errlSeverity_t
-{
-    FAPI2_ERRL_SEV_UNDEFINED     = 0x00, /// Used internally by ffdc mechanism
-    FAPI2_ERRL_SEV_RECOVERED     = 0x10, /// Not seen by customer
-    FAPI2_ERRL_SEV_PREDICTIVE    = 0x20, /// Error recovered but customer will see
-    FAPI2_ERRL_SEV_UNRECOVERABLE = 0x40  /// Unrecoverable, general
+// taken from error_info_defs.H, including the header file breaks logging and
+// debug collector as they need to include ekb in include path
+enum errlSeverity_t {
+	FAPI2_ERRL_SEV_UNDEFINED = 0x00, /// Used internally by ffdc mechanism
+	FAPI2_ERRL_SEV_RECOVERED = 0x10, /// Not seen by customer
+	FAPI2_ERRL_SEV_PREDICTIVE =
+	    0x20, /// Error recovered but customer will see
```

Comment:
Why line break for this line when others don't have? Also I guess some tab/space issue here